routes/teams.js

```diff
@@ -0,0 +1,162 @@
+const express = require('express');
+const router = express.Router();
+const { pool } = require('../config');
+
+const sendError = (res) =>
+  res.status(400).json({ status: 'error', message: 'An error occured' });
+
+const sendSuccess = (res, code, message) =>
+  res.status(code).json({ status: 'success', message });
+
+const getTeam = (req, res) => {
+  const { teamId } = req.params;
+
+  pool.query(
+    'SELECT U.id, U.name FROM users AS U, team_members AS M WHERE U.id=M.user_id AND M.team_id=$1',
+    [teamId],
+    (error, results) => {
+      if (error) {
+        sendError(res);
+        return;
+      }
+      res.status(200).json(results.rows);
+    }
+  );
+};
+
+const getAllTeams = (req, res) => {
+  pool.query('SELECT * FROM teams', (error, results) => {
+    if (error) {
+      sendError(res);
```

Comment:
Missing return statement here. If there's an error it will send an error response then try to send a success.